src/python/twitter/common/fs/hdfs.py

```diff
@@ -52,10 +57,14 @@ def config(self):
     return self._config
 
   def _call(self, cmd, *args, **kwargs):
-    """Runs hdfs fs command  with the given command and args.
+    """Runs fs command with the given command and args.
     Checks the result of the call by default but this can be disabled with check=False.
     """
-    cmd = ['hdfs', '--config', self._config, 'dfs', cmd] + list(args)
+    self._hadoop_command = 'hdfs'
+    if self.use_hadoop_v1 or self._cmd_class.execute_suppress_stdout_stderr('hdfs') != 0:
```

Comment:
This is breaking our recent build, use_hadoop_v1 does not seem to be a class member
```python
self._use_hadoop_v1 = use_hadoop_v1
```
seems to be missing initialization